src/components/DatetimePicker/DatetimePicker.vue

```diff
@@ -57,15 +58,15 @@
 </template>
 
 <script>
-import DatePicker from 'vue2-datepicker/lib/datepicker'
+import DatePicker from 'vue2-datepicker'
 
 /**
  * remove leading zeros on hours and minutes
  * https://github.com/mengxiong10/vue2-datepicker/blob/65c5762227649430f14158c01401a8486a881336/src/panel/time.js#L38
  */
-DatePicker.components.CalendarPanel.components.PanelTime.methods.stringifyText = function(data) {
```

Comment:
I tend to just drop that, leading zeros are common in times and i don't think such a change justifies hacks like that.

Reply:
It was Jan's request ;)

Reply:
Let me quickly fix this as well.